external/sentry.go

```diff
@@ -0,0 +1,31 @@
+package external
+
+import (
+	"github.com/getsentry/sentry-go"
+	log "github.com/sirupsen/logrus"
+	"golbat/config"
+	"time"
+)
+
+const sentryDsn = "https://bc0b5dc992ee486a90faa20b83016351@sentry.mew.is/2"
+
+func InitSentry() {
+	if !config.Config.Sentry.Disable {
```

Comment:
negative config verbs are horrible. Should be enable.
Should it be enabled by default?

Reply:
I agree, change config Variable to ebable, yes it's enabled by default

Reply:
Would definitely speed up error's lookup. But it depends, I can keep it disabled by default. I'll rework to `Enable`

Reply:
You can make it enable and enable it by default by setting to true before loading config. But this would need to be very clear to users in readme

Reply:
No need. Dropping idea of default DSN.